desktop/packages/linux/rs.ruffle.Ruffle.metainfo.xml

```diff
@@ -4,21 +4,33 @@
     <metadata_license>CC0-1.0</metadata_license>
     <project_license>MIT OR Apache-2.0</project_license>
     <name>Ruffle</name>
+    <name xml:lang="ar">رَفل</name>
+    <name xml:lang="cs">Ruffle</name>
+    <name xml:lang="es">Ruffle</name>
     <name xml:lang="fr">Ruffle</name>
+    <name xml:lang="ja">Ruffle</name>
     <name xml:lang="ko">Ruffle</name>
     <name xml:lang="nl">Ruffle</name>
     <name xml:lang="pl">Ruffle</name>
+    <name xml:lang="ru">Ruffle</name>
     <name xml:lang="zh-CN">Ruffle</name>
     <summary>Play Flash games &amp; movies</summary>
+    <summary xml:lang="ar">شغِّل ألعاب وأفلام الفلاش</summary>
+    <summary xml:lang="es">Jugar juegos &amp; ver películas de Flash</summary>
```

Comment:
Again, this should be the same as in the desktop file, here it has an ampersand.

Reply:
Commented on Crowdin